index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Javascript Documentation</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+<div class="row">
+
+<!-------- Left Column--------->
+    <div class="col1">
+        <nav id="navbar">
+            <header><h1>JS Documentation</h1></header><br><hr>
+            <a href="#introduction" class="nav-link">Introduction</a><br><br><hr>
+
+<a href="#what_you_should_already_know" class="nav-link">What you should already know</a><br><br><hr>
+
+<a href="#javascript_and_java" class="nav-link">Javascript and Java</a><br><br><hr>
+
+<a href="#hello_world" class="nav-link">Hello World</a><br><br><hr>
+
+<a href="#variables" class="nav-link">Variables</a><br><br><hr>
+
+<a href="#declaring_variables" class="nav-link">Declaring variables</a><br><br><hr>
```

Comment:
I would like to suggest that the hyperlinks be placed inside ul > li to keep with industry's best practices.

Reply:
Also, as a re-iteration of the first comment, I would still recommend using white spaces with care.